scene/2d/tile_map.cpp

```diff
@@ -977,6 +977,14 @@ void TileMap::set_cell_autotile_coord(int p_x, int p_y, const Vector2 &p_coord)
 	c.autotile_coord_x = p_coord.x;
 	c.autotile_coord_y = p_coord.y;
 	tile_map[pk] = c;
+
```

Comment:
I had to update this because tiles weren't drawing correctly in the editor if only the auto tile position changed, so I had to mark them as dirty too.